app/controller/NavigationController.js

```diff
@@ -49,18 +49,18 @@ SDL.NavigationController = Em.Object.create(
      * @param {Object} request
      */
     sendLocation: function(request) {
-      this.model.LocationDetails.push(
+      this.model.get('LocationDetails').addObject(

```

Comment:
@YarikMamykin try simply this
```suggestion
    this.model.LocationDetails.pushObject(
```

Reply:
@AKalinich-Luxoft fixed in https://github.com/LuxoftSDL/sdl_hmi/pull/7/commits/277299046a476f95105d9c4ca0c3ca231377968c